.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "presets": [
-    ["es2015", { "modules": false }],
```

Comment:
with `{ modules: false}` we define not to transform modules to commonjs. https://babeljs.io/docs/plugins/preset-es2015/#optionsmodules. What was the motivation to change to commonjs?